test/docfx.Snapshot.Tests/SamplesTest.CSharp/api/CSharp8.ReadOnlyMembers.html.view.verified.json

```diff
@@ -989,7 +990,7 @@
           },
           "level": 0.0,
           "type": "method",
-          "summary": "",
+          "summary": "<p sourcefile=\"api/CSharp8.ReadOnlyMembers.yml\" sourcestartlinenumber=\"1\">Returns the fully qualified type name of this instance.</p>\n",
```

Comment:
Looks like `ValueType.ToString` now gets a default XML comment documenting the return type is FQTN. Overriding the `ToString` method of a struct *should* also override the XML documentation.